tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/components/forms/button/.content.xml

```diff
@@ -3,15 +3,32 @@
   xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
   xmlns:ws="http://ds.pl/websight"
   xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="ws:Page">
-  <jcr:content jcr:primaryType="ws:PageContent" jcr:title="Button" sling:resourceType="typerefinery/components/structure/page" ws:template="/apps/typerefinery/templates/page">
+  <jcr:content jcr:primaryType="ws:PageContent" jcr:title="Button's" sling:resourceType="typerefinery/components/structure/page" ws:template="/apps/typerefinery/templates/page">
     <rootcontainer jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/fixedrootcontainer">
       <header jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/header"/>
       <maincontainer jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/maincontainer">
-        <pagesection jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/pagesection">
-          <title jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/content/title" title="Button Component"/>
-          <button jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/button"/>
-        </pagesection>
+        <container1 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/container">
+          <container1_1 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/container">
```

Comment:
can you have them inside the form container ?
why are we using container for form components ?

Reply:
form container added.